pkg/api/routes/components_test.go

```diff
@@ -43,6 +45,19 @@ var validComponents = fleetdb.ServerComponentSlice{
 	},
 }
 
+var validComponentTypes = fleetdb.ServerComponentTypeSlice{
```

Comment:
I don't understand why we're testing this? Is this related to my comment about the FleetDBClient constructor?

Reply:
Srry this is rebase issue. Already added in the last PR.